### DISCUSSION TITLE
SVG Sprite vs SVG Component Performance

### DISCUSSION THREAD

**Discussion by HasanHaghniya** [vercel/next.js#14312] [Ideas]
hey next community 

i have about 70 SVG icons , for performance which way do you prefer to keep them? as a SVG Sprite or make component for every icons? also number of requests 

Thanks

**Comment by LedzZm** [comment]
Bump this.

With the power of svgs it would be great, if we could have an article or discussion about best practices in how to use them in nextjs projects.

Are there any guidelines at the moment about how to use a sprite? Any input would be appreciated (Y)

**Reply by Emiliano-Bucci** [reply]
I normally use sprites, using the following package -> https://github.com/cyrilwanner/next-optimized-images I believe in case like these, `sprites` are better from performance perspective.

**Reply by LedzZm** [reply]
could you share your webpack configuration? 

I also stumbled upon this package but it seems discontinued in general after the Next.js 10 image optimizations.

Could it be the case, that there are plans to incorporate svg sprites to next core image optimizations?

Thanks again :)

**Reply by LedzZm** [reply]
I see they provide a more lightweight solution 


// next.config.js
const withOptimizedImages = require('next-optimized-images');

module.exports = withOptimizedImages({
  /* config for next-optimized-images */

  // your config for other plugins or the general next.js here...
}); 

But I am not sure how the svgo configurations etc should be incorporated.
